sos/report/plugins/postfix.py

```diff
@@ -17,6 +17,43 @@ class Postfix(Plugin):
 
     packages = ('postfix',)
 
+    def forbidden_ssl_keys_files(self):
+        # list of attributes defining a location of a SSL key file
+        # we must forbid from collection
+        forbid_attributes = [
+            "lmtp_tls_dkey_file",
+            "lmtp_tls_eckey_file",
+            "lmtp_tls_key_file",
+            "smtp_tls_dkey_file",
+            "smtp_tls_eckey_file",
+            "smtp_tls_key_file",
+            "smtpd_tls_dkey_file",
+            "smtpd_tls_eckey_file",
+            "smtpd_tls_key_file",
+            "tls_legacy_public_key_fingerprints",
+            "tlsproxy_tls_dkey_file",
+            "tlsproxy_tls_eckey_file",
+            "tlsproxy_tls_key_file",
+            "smtpd_tls_dh1024_param_file",
+            "smtpd_tls_dh512_param_file",
+            "tlsproxy_tls_dh1024_param_file",
+            "tlsproxy_tls_dh512_param_file",
+        ]
+        fp = []
+        try:
+            cffile = open('/etc/postfix/main.cf', 'r')
+            lines = cffile.readlines()
+            cffile.close()
+            for line in lines:
+                # ignore comments and empty lines
+                # treat only lines like
+                # lmtp_tls_dkey_file = /path/to/file
+                words = line.split('#')[0].split()
+                if (len(words) == 3) and (words[0] in forbid_attributes):
+                    fp.append(words[2])
```

Comment:
We could split on the `=` and side-step potential formatting problems in terms of whitespace and length:

```python3
with open('/etc/postfix/main.cf', 'r') as cffile:
    for line in cffile.readlines():
        if line.startswith('#'):
            continue
        words = line.split('=')
        if words[0] in forbid_attributes:
            fp.append(words[1])
```